# Fix compilation error: multiple definition of `editline_history_file'
# The variable is defined in both editline.c and siodeditline.c
# Author: Chia-Jung Hsu, 2024-03-19
diff --git a/siod/editline.c b/siod/editline.c
index f6eb5c6..d6163d8 100644
--- a/siod/editline.c
+++ b/siod/editline.c
@@ -73,7 +73,7 @@
 /* modified by awb to allow specifcation of history size at run time  */
 /* (though only once)                                                 */
 int editline_histsize=256;
-char *editline_history_file;
+// char *editline_history_file;
 /* If this is defined it'll be called for completion first, before the */
 /* internal file name completion will be                               */
 EL_USER_COMPLETION_FUNCTION_TYPE*el_user_completion_function = NULL;

